atom/index.js

```diff
@@ -1,81 +1,93 @@
 import { clean } from '../clean-stores/index.js'
-
-let listenerQueue = []
+import { getStoreState, globalContext } from '../context/index.js'
 
 export let atom = (initialValue, level) => {
-  let listeners = []
   let $atom = {
+    ctx: globalContext,
```

Comment:
_Sorry, I will have dump question. It is a trade-off of quick review (like SAW vs DOM parsers)_

Why do we need to bind global context to every context?

Reply:
I suppose we could, but it'll be in more places. Currently `this.ctx` appears 4 times, so this is just 4 times repeated the same `.ctx || globalContext`. Seems easier to me to just add context to all stores by default. Do you see any downsides?